src/sortStudents.ts

```diff
@@ -1,16 +1,83 @@
-
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
+}
+
+export type SortOrder = 'asc'|'desc';
+
+function findAverage(grades:number[]):number {
+  return grades.reduce((prev, grade) => prev + grade) / grades.length;
 }
 
-// create SortOrder type
-export type SortOrder;
+export function sortStudents(
+  students:Student[],
+  sortBy: SortType,
+  order:SortOrder,
+):object[] {
```

Comment:
```suggestion
  students: Student[],
  sortBy: SortType,
  order: SortOrder,
): object[] {
```

let's keep formatting consistent